teleport/tests/conftest.py

```diff
@@ -0,0 +1,25 @@
+# (C) Datadog, Inc. 2024-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import os
+
+import pytest
+
+from datadog_checks.dev import docker_run, get_here
+
+
+@pytest.fixture(scope='session')
+def dd_environment():
+    compose_file = os.path.join(get_here(), 'docker', 'docker-compose.yaml')
+
+    def nothing():
```

Comment:
What's the reason for this?

Reply:
I was experimenting with using the same docker container for longer to be able to collect all the metrics we needed, since some metrics take longer to be generated.
It's no longer needed, I'll remove it.